gwcs/geometry.py

```diff
@@ -74,68 +76,134 @@ class SphericalToCartesian(Model):
     n_inputs = 2
     n_outputs = 3
 
-    def __init__(self, **kwargs):
+    def __init__(self, wrap_lon_at=360, **kwargs):
+        """
+        Parameters
+        ----------
+        wrap_lon_at : {360, 180}, optional
+            An **integer number** that specifies the range of the longitude
+            (azimuthal) angle. When ``wrap_lon_at`` is 180, the longitude angle
+            will have a range of ``[-180, 180)`` and when ``wrap_lon_at``
+            is 360 (default), the longitude angle will have a range of
+            ``[0, 360)``.
+
+        """
         super().__init__(**kwargs)
-        self.inputs = ('phi', 'theta')
+        self.inputs = ('lon', 'lat')
```

Comment:
This is technically a breaking change, right?  Since previous users would have been able to do this:

```python
model_instance(phi=80, theta=60)
```

Reply:
Not really. This model was available only for a couple of days _and_, most importantly, `gwcs` has not yet been released.